spynnaker/pyNN/models/neuron/population_machine_vertex.py

```diff
@@ -47,13 +47,27 @@ class PopulationMachineVertex(
 
     def __init__(
             self, resources_required, is_recording, minimum_buffer_sdram_usage,
-            buffered_sdram_per_timestep, label, constraints=None):
+            buffered_sdram_per_timestep, label, constraints=None,
```

Comment:
If get_buffered_sdram_per_timestep is changed to get_n_timesteps_in_buffer_space, this should take the neuron_recorder instead of buffered_sdram_per_timestep and overflow_sdram.